### PR TITLE
(#862) Add CCM timezone information

### PR DESCRIPTION
**Pull request by corbob**
## Description Of Changes

Add in details about timezones within Chocolatey Central Management. Also fills out the General Settings page in Chocolatey Central Management.

Note: This still needs images. Our internal tracking of images has been updated.

## Motivation and Context

Updating Chocolatey Central Management documentation before the upcoming release.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [x] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [x] Requires a change to menu structure (top or left hand side)/
* [x] Menu structure has been updated

## Related Issue

Related to #862

### CODE REVIEW

File: input/en-us/central-management/faq.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Timezone FAQs
+
+### Why Doesn't Chocolatey Central Management Have My Timezone?
+
+Chocolatey Central Management uses the Windows install of its host to determine if a timezone is valid. If Windows does not know of a timezone, then it will not be made available.
+
+Some examples of timezones that may not appear:
+
+* `Qyzylorda Standard Time` does not appear in Windows Server 2016 until you install Windows Update 5012170 or later.
+* `Yukon Standard Time`does not appear in Windows Server 2016 until you install 5031462 or later.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/faq.md:16]:
```suggestion
* `Yukon Standard Time` does not appear in Windows Server 2016 until you install 5031462 or later.

```

Also, I assume the mention of 5031462 is a Windows Update?
This should probably be made clearer.

File: input/en-us/central-management/faq.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* `South Sudan Standard Time` does not appear in Windows Server 2016 until you install 5031462 or later.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/faq.md:17]: Same on this one. Need to call out what 5031462 is.

File: input/en-us/central-management/faq.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Why Did I Get a Notification That My Chosen Timezone is Invalid?
+
+Starting in Chocolatey Central Management version 0.12.0, if the chosen timezone is detected to be invalid, we will set it back to the default timezone and alert you through a notification.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/faq.md:21]: This implies that if the user has an invalid time zone set, they will be notified.

What if the global configuration for CCM Website is invalid, who will be notified in this case?

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/faq.md (hunk 3 above)
Comment (corbob) [reply input/en-us/central-management/faq.md]: Ah, I had intended to update this section, but was running into some Docker fun and apparently forgot. I've updated this whole section to hopefully be more clear.

**Review by AdmiringWorm — DISMISSED** [review]
LGTM, only thing left is to get the image added.

File: input/en-us/central-management/usage/website/administration/settings/general.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ Title: General
 Description: Information about using up general settings within the Administration Settings screen
 ---
 
-> :choco-warning: **WARNING**
->
-> This is a Work in Progress. Please check back later.
\ No newline at end of file
+The Chocolatey Central Management Administration Settings page loads the General tab by default. This page is used to set the Chocolatey Central Management default timezone.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/administration/settings/general.md:8]:
```suggestion
The Administration Settings page loads the General tab by default. This page is used to set the default timezone.
```

File: input/en-us/central-management/usage/website/administration/settings/general.md [hunk 5]
```diff
@@ -5,6 +5,8 @@ Title: General
...
+
+If a user does not set their own timezone for display from `My Settings`, then the selected timezone will be used for the display of all times within Chocolatey Central Management.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/administration/settings/general.md:10]:
```suggestion
If a user does not set their own timezone for display from `My Settings`, then the selected timezone will be used for the display of all times.
```

File: input/en-us/central-management/faq.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+* `Qyzylorda Standard Time` does not appear in Windows Server 2016 until you install Windows Update [5031362](https://support.microsoft.com/en-us/topic/october-10-2023-kb5031362-os-build-14393-6351-0c6e713e-3d6a-4593-8a75-af0a605f249c) or later.
+* `Yukon Standard Time` does not appear in Windows Server 2016 until you install Windows Update [5031362](https://support.microsoft.com/en-us/topic/october-10-2023-kb5031362-os-build-14393-6351-0c6e713e-3d6a-4593-8a75-af0a605f249c) or later.
+* `South Sudan Standard Time` does not appear in Windows Server 2016 until you install [5031362](https://support.microsoft.com/en-us/topic/october-10-2023-kb5031362-os-build-14393-6351-0c6e713e-3d6a-4593-8a75-af0a605f249c) or later.
+
+### Why Did I Get a Notification That My Chosen Timezone is Invalid?
+
+Starting in Chocolatey Central Management version 0.12.0, if the chosen timezone is detected to be invalid, we will set it back to the default timezone and alert you through a notification. If you are a Chocolatey Central Management administrator, you may additionally receive a notification that the Chocolatey Central Management timezone was invalid.
```
Comment (gep13) [inline input/en-us/central-management/faq.md:21]:
```suggestion
Starting in Chocolatey Central Management version 0.12.0, if the chosen timezone is detected to be invalid, we will set it back to the default timezone and alert you through a notification. If you are an Administrator, you may additionally receive a notification that the system timezone was invalid.

```

File: input/en-us/central-management/faq.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+#### User Account Timezone is Invalid
+
+If you received the below notification, then it means that the timezone used for your user account was invalid, and has been set back to the default used by Chocolatey Central Management. You can pick your own default by clicking the user menu in the top right and selecting `My Settings`
```
Comment (gep13) [inline input/en-us/central-management/faq.md:25]:
```suggestion
If you received the below notification, then it means that the timezone used for your user account was invalid, and has been set back to the default timezone. You can pick your own default by clicking the user menu in the top right and selecting `My Settings`

```

File: input/en-us/central-management/faq.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+![Example notification for a user configured timezone being invalid](/assets/images/ccm/administration/settings/user-timezone-invalid-notification.png)
+
+
+#### Chocolatey Central Management Timezone is Invalid
+
+If you received the below notification, then it means that the timezone used for Chocolatey Central Management was invalid, and has been set back to the default of `UTC`. You can adjust the Chocolatey Central Management default through the [Settings page](xref:ccm-administration-settings-general).
```
Comment (gep13) [inline input/en-us/central-management/faq.md:32]:
```suggestion
If you received the below notification, then it means that the system timezone was invalid, and has been set back to the default of `UTC`. You can adjust the default through the [Settings page](xref:ccm-administration-settings-general).

```

**Review by gep13 — CHANGES_REQUESTED** [review]

**Comment by corbob** [comment]
@gep13 I've merged in your suggestions for some of the changes, but I'm hesitant to merge the last two that would refer to the system timezone. I was specifically trying to avoid the term "system" here as the "system" is ambiguous. It could refer to the system running CCM, or the system the user is accessing CCM from.

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**